services/web/src/App.tsx

```diff
@@ -1,42 +1,50 @@
-import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
-import './App.css';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@radix-ui/react-accordion';
+'use client';
 
-import MockDataComponent from './website/mockComponent';
```

Comment:
Recover the old App.tsx and add the input. I had put a sample request component previously which will be useful later for you to see how it works.